docs/root/start/start.rst

```diff
@@ -0,0 +1,154 @@
+.. _start:
+
+Getting Started
+===============
+
+This section gets you started with a very simple configuration, and provides some example configurations.
+
+Envoy does not currently provide separate pre-built binaries, but does provide Docker images. This is
+the fastest way to get started using Envoy. Should you wish to use Envoy outside of a
+Docker container, you will need to :ref:`build it <building>`.
+
+These examples use the :ref:`v2 Envoy API <envoy_api_reference>`, but use only the static configuration
+feature of the API, which is most useful for simple requirements. For more complex requirements
+:ref:`Dynamic Configuration <arch_overview_dynamic_config>` is supported.
+
+Quick Start to Run Simple Example
+---------------------------------
+
+These instructions run from files in the Envoy repo. The sections below give a
+more detailed explanation of the configuration file and execution steps for
+the same configuration.
+
+A very minimal Envoy configuration that can be used to validate basic plain HTTP
+proxying is available in :repo:`configs/google_com_proxy.v2.yaml`. This is not
+intended to represent a realistic Envoy deployment.
+
+Copy both :repo:`configs/Dockerfile` and
+:repo:`configs/google_com_proxy.v2.yaml` to the same directory on your local
+disk. Then, build and run the Dockerfile, and test out Envoy by sending a
+request to port 10000::
+
+  $ docker build -t envoy-google-test:v1 .
+  $ docker run -d -p 10000:10000 envoy-google-test:v1
+  $ curl -v localhost:10000
+
+The Dockerfile will build a container containing the latest version of Envoy,
+and copy a basic Envoy configuration into the container. This basic
+configuration tells Envoy to route incoming requests to \*.google.com.
+
+
+Simple Configuration
+--------------------
+
+Envoy can be configured using a single YAML file passed in as an argument on the command line.
+
+The :ref:`admin message <envoy_api_msg_config.bootstrap.v2.Admin>` is required to configure
+the administration server. The `address` key specifies the :ref:`address <envoy_api_file_envoy/api/v2/core/address.proto>`
+which in this case is simply `localhost:9901`.
+
+.. code-block:: yaml
+
+  admin:
+    access_log_path: /tmp/admin_access.log
+    address:
+      socket_address: { address: 0.0.0.0, port_value: 9901 }
+
+The :ref:`static_resources <envoy_api_field_config.bootstrap.v2.Bootstrap.static_resources>` contains everything that is configured staticaly when Envoy starts,
+as opposed to the means of configuring resources dynamically when Envoy is running.
+
+.. code-block:: yaml
+
+    static_resources:
+
+The :ref:`listeners <envoy_api_file_envoy/api/v2/listener/listener.proto>` blah.
+
+.. code-block:: yaml
+
+      listeners:
+      - name: listener_0
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 10000 }
+        filter_chains:
+        - filters:
+          - name: envoy.http_connection_manager
+            config:
+              stat_prefix: ingress_http
```

Comment:
Do we want a comment about each field here describing what is roughly meant?

Reply:
I think that will be very helpful. Ex this is the http l7 filter that can provide and for routing domain ‘*’ will match all requests on on this port.

Reply:
Yes, this is part of the WIP, I did not finish this section, and I will do it at a detailed level going through each field (or collection of fields, as it makes sense).